pom.xml

```diff
@@ -93,6 +94,11 @@
             <name>Nexus Releases</name>
             <url>https://nexus.internal.fairdatapoint.org/repository/maven-releases/</url>
         </repository>
+        <repository>
+            <id>github</id>
+            <name>GitHub Packages</name>
+            <url>https://public:&#103;hp_QKa2ikVLWK6EN0mFGH2PlA6Ctz1MNr0WkkK7@maven.pkg.github.com/fairdevkit/rdf-resource-resolver</url>

```

Comment:
Why is "public" and password/token here? Isn't that public and could work like this https://github.com/TobseF/github-plugin-registry-example/blob/master/pom.xml#L15?